swingset/src/main/java/com/nqadmin/swingset/SSDataNavigator.java

```diff
@@ -363,6 +400,52 @@ public SSDataNavigator(final RowSet _rowSet, final Dimension _buttonSize) {
 		addToolTips();
 		createPanel();
 		addNavListeners();
+
+		// setSSRowSet will typically set the eventBus
+		setupEventBus();
+	}
+
+	// TODO: Is it necessary to replace event bus when something changes?
+	//       getLocalEventBus doesn't look at rowSet (or this for that matter)
+	private void setupEventBus() {
+// TODO: ENABLE FOR EVENTBUS
+//		if (eventBus == null) {
+//			eventBus = getLocalEventBus(this, rowSet);
+//		}
+//		eventBus.register(new BusReceiver());
+	}
+
+	// TODO: also have Set<SSComponentInterface> modifiedComponents
+	private final Set<SSComponentInterface> errorComponents = new HashSet<>();
+	class BusReceiver {
+// TODO: ENABLE FOR EVENTBUS		
+//		@Subscribe
+//		public void handleRowDataChanged(RowSetModificationEvent ev) {
```

Comment:
Not that it matters, but I think you could just have just commented out the `@Subscribe`